x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/time_range_step/time_range_picker.tsx

```diff
@@ -56,7 +56,7 @@ export const TimeRangePicker: FC<Props> = ({ setTimeRange, timeRange }) => {
 
   return (
     <Fragment>
-      <div style={{ minWidth: WIDTH }}>
+      <div style={{ minWidth: WIDTH }} data-test-subj={`jobWizardDateRange`}>
```

Comment:
I see most `data-test-subj` have a `ml` prefix but this one does not - should we generally have it?

Reply:
Good question. When I started creating test subjects, I've named them `ml*` with a rather long descriptive name, but over time I began to give shorter names to make parent-child relations better to read, e.g. ```testSubjects.getVisibleText(`dataSplit > splitCard front > splitCardTitle`)```. I'm still not perfectly sure what the best way of naming is, but `ml` namespacing is probably a good idea, so I will go through our test subjects in a follow-up and add it where needed.

Reply:
Addressed in  #47032